api/nodes/views.py

```diff
@@ -276,8 +277,12 @@ class NodeList(JSONAPIBaseView, bulk_views.BulkUpdateJSONAPIView, bulk_views.Bul
 
     # overrides NodesFilterMixin
     def get_default_queryset(self):
-        user = self.request.user
-        return default_node_list_queryset() & default_node_permission_queryset(user)
+        # this is an example of using a flag inside an API view to enable/disable
+        if waffle.flag_is_active(self.request, 'api_flag_on_user_id'):
```

Comment:
It'd be convenient to have a decorator for this use case:

```python
@require_flag('api_flag_on_user_id')
def get_default_queryset(self):
    user = self.request.user
     #....
```

Reply:
good idea, I've added some decorators with example usages.